docs/APIReference-Route.md

```diff
@@ -70,6 +76,57 @@ class ProfileRoute extends Relay.Route {
 }
 ```
 
+### prepareParams (static property)
+
+```
+static prepareParams: ?(prevParams: {[prevParam: string]: mixed}) => {[param: string]: mixed};
+```
+
+Routes can declare additional parameters, or conversion for parameters on initialisation. These can also be used as query parameters in your nested containers.
+
+#### Example
+
+```
+class ProfileRoute extends Relay.Route {
+  static queries = {
+    viewer: () => Relay.QL`query { viewer }`
+  };
+  static prepareParams = (prevParams) => {
+    return {
+      // Ensure any params that ProfileRoute gets initialised with will still be passed down.
+      ...prevParams,
+      //  Override the `limit` variable in the top level container.
+      limit: 10
+    }
+  }
+  // ...
+}
+
+const Container = Relay.createContainer(({viewer}) => (
+  <View>
+    {viewer.edges.map((node) => <div>node.name</div>)}
```

Comment:
Thanks for the catch. The one disadvantage of being fullstack and not having been able to write any meaningful JS in about a month.